dbcog/monster_graph.py

```diff
@@ -48,17 +49,19 @@
   active_skills{0}.desc_ko AS as_desc_ko,
   active_skills{0}.turn_max,
   active_skills{0}.turn_min,
-  series.name_ja AS s_name_ja,
-  series.name_en AS s_name_en,
-  series.name_ko AS s_name_ko,
-  series.series_type AS s_series_type,
+  COALESCE(series.series_id, 0) AS s_series_id,
+  COALESCE(series.name_ja, 'Unsorted') AS s_name_ja,
+  COALESCE(series.name_en, 'Unsorted') AS s_name_en,
+  COALESCE(series.name_ko, 'Unsorted') AS s_name_ko,
+  COALESCE(series.series_type, NULL) AS s_series_type,
   exchanges.target_monster_id AS evo_gem_id,
   drops.drop_id
 FROM
   monsters{0}
   LEFT OUTER JOIN leader_skills{0} ON monsters{0}.leader_skill_id = leader_skills{0}.leader_skill_id
   LEFT OUTER JOIN active_skills{0} ON monsters{0}.active_skill_id = active_skills{0}.active_skill_id
-  LEFT OUTER JOIN series ON monsters{0}.series_id = series.series_id
+  LEFT OUTER JOIN monster_series ON monsters{0}.monster_id = monster_series.monster_id AND priority = 1
+  LEFT OUTER JOIN series ON monster_series.series_id = series.series_id
```

Comment:
should monster_name_overrides be in this table list?

Reply:
It's the last join